mlflow/server/js/src/components/PermissionDeniedView.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import overflow from '../static/404-overflow.svg';
+import Colors from '../styles/Colors';
+
+export default class PermissionDeniedView extends Component {
+  static propTypes = {
+    errorMessage: PropTypes.string.isRequired,
+  };
+
+  render() {
+    return (
+      <div>
+        <img
+          className='center'
+          alt="404 Not Found"
+          style={{ height: '300px', marginTop: '80px' }}
+          src={overflow}
+        />
+        <h1 className="center" style={{ paddingTop: '10px' }}>
+          Permission Denied
+        </h1>
+        <h2 className="center" style={{ color: Colors.secondaryText }}>
+          {this.props.errorMessage || 'The current does not have permission to view this page.'}
```

Comment:
"The current" >> "The current user" (???)

Reply:
Yeah that does sound better :).